src/zoo/Runner.java

```diff
@@ -14,7 +14,13 @@ public static void main(String[] args) {
         zoo.add(dog);
         zoo.add(tiger);
 
-
+//        first version:
+        for(int i = 0; i < zoo.size(); i++){
+            Animal animal = zoo.get(i);
+            System.out.println(animal.getClass().getSimpleName() +" gives voice: " + animal.giveVoice());
```

Comment:
why `getClass.getSimpleName` ;)? we have `toString` method for such use cases. We rarely should need the class's name. It's checking the type whereas in object oriented programming, the type shouldn't be checked very often. We will learn polymorphism and we'll learn that a `.` is more important than the type :)

Reply:
ok, done:)